felix/bpf/ut/map_upgrade_test.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
```

Comment:
i think this tests should be next to felix/bpf/maps.go, WDYT? felix/bpf/ut is kinda for unittesting the programs / dataplane

Reply:
make ut doesn't run a privileged container. Hence I am not able to run these tests as part of it. ut-bpf looks to be the right place.